tests/crash_recovery-test.cc

```diff
@@ -10,9 +10,18 @@
 int               crashes = 0;
 std::atomic<bool> expectFault;
 
+static void test_irqs_are_enabled()
+{
+	void *r = __builtin_return_address(0);
+	TEST(__builtin_cheri_type_get(r) == CheriSealTypeReturnSentryEnabling,
```

Comment:
For future reference: we have `TEST_EQUAL` which gives better error messages in this situation.